src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,258 @@
+                    )
+                )
+
+                // Obtain the color of the circle
+                const combinedColor = this.primeFactors(
+                    Number(number_now),
+                    Number(this.count_prime)
+                )
+
+                this.sketch.colorMode(this.sketch.HSB)
+                this.sketch.fill(combinedColor, 100, bright)
+                this.sketch.ellipse(this.X, this.Y, radius, radius)
+
+                // Change brightness regarding
+                //  the difference between 1/(number)^n and 1/(number)^(n-1)
+                bright = this.changeColor(diff, bright)
+
+                radius -= 1
+            }
+
+            this.sketch.fill('white')
+            this.sketch.text(
+                '1/'.concat(String(number_now)).concat('^n'),
+                this.X,
+                this.Y
+            )
+            this.X += 100
+            if (this.X >= 800) {
+                this.X = 800 / this.n + 30
+                this.Y += 100
+            }
+
+            if (this.current_num >= this.n) {
+                this.firstDraw = false
+            }
+            this.sketch.noStroke()
+        } else {
+            //Bug: multiple clicks detected when only one click happened
+            //Show description box when the lft arrow key is pressed
+            if (this.sketch.keyIsDown(this.sketch.LEFT_ARROW)) {
+                console.log(countClick++)
+                if (this.boxIsShow == true && this.subG != null) {
+                    this.boxIsShow = false
+                    this.undrawBox()
+                } else {
+                    this.boxIsShow = true
+                    this.drawBox()
+                }
+            }
+            // Refresh
+            if (
+                this.sketch.keyIsDown(this.sketch.RIGHT_ARROW)
+                && this.firstDraw == false
+            ) {
+                this.firstDraw = true
+            }
+        }
+    }
+
+    drawBox() {
+        //Create a white background for the description box
+        this.subG = this.sketch.createGraphics(800, 90)
+        this.subG.colorMode(this.sketch.HSB)
+        this.subG.noStroke()
+        this.subG.fill(0, 0, 100)
+        this.subG.rect(0, 0, 800, 90)
+        this.subG.fill('black')
+        let tmpX = 0
+        let tmpY = 0
+
+        //Show the color of every prime number
+        for (let i = 0; i < this.primeNum.length; i += 1) {
+            this.subG.fill('black')
+            this.subG.text(String(this.primeNum[i]), 10 + tmpX, 15 + tmpY)
+            this.subG.fill(colorMap.get(this.primeNum[i]), 100, 100)
+
+            this.subG.ellipse(35 + tmpX, 15 + tmpY, 10, 10)
+            tmpX += 50
+            if (tmpX >= 800) {
+                tmpX = 0
+                tmpY += 30
+            }
+        }
+
+        this.sketch.image(this.subG, 0, 700)
+
+        //this.sketch.noLoop()
+    }
+
+    undrawBox() {
+        if (this.subG) {
+            console.log('undraw')
```

Comment:
`console.log()` is great for development, but when we deploy our front-end code, it's generally considered unprofessional to have logs in the console.